invenio_communities/models.py

```diff
@@ -240,6 +256,20 @@ def get(cls, community_id, with_deleted=False):
         return q.one_or_none()
 
     @classmethod
+    def get(cls, community_id=None, community_name=None, with_deleted=False):
+        """Get a community."""
+        assert((community_id is not None) ^ (community_name is not None),
```

Comment:
Wouldn't this make the API even more strange as this gets multiple communities when the community_name and community_id don't match the same Communities?
If I am not mistaken every other `get` method in Invenio APIs returns one object max.